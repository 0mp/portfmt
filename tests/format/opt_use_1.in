QT5_USE=	QT=buildtools_build,widgets,core,gui,xml,qmake_build
X11_USE=	xorg+=x11,bitmaps,ice
