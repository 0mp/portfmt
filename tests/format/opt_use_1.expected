QT5_USE=	QT=core,gui,widgets,xml,buildtools_build,qmake_build
X11_USE=	XORG+=bitmaps,ice,x11
